### PR TITLE
Feature/kotlin dsl

### PR DESCRIPTION
**Pull request by DRSchlaubi**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+tasks {
+    compileJava {
+        options.encoding = "UTF-8"
+    }
+}
+
+dependencies {
+    // add your dependencies here
```
Comment (freyacodes) [inline build.gradle.kts:29]: An example here would be good, particularly as it's not clear which dependency configuration to use

Comment (DRSchlaubi) [reply build.gradle.kts:29]: You use the one you would use per gradle convention `implementation` or `compileOnly`

Comment (DRSchlaubi) [reply build.gradle.kts:29]: So not really sure for what the example should be

Comment (freyacodes) [reply build.gradle.kts:29]: I suppose

**Review by freyacodes — APPROVED** [review]

**Review by topi314 — APPROVED** [review]

**Pull request merged**